### ISSUE TITLE
[Accessibility], WCAG 2.1.1, 2.4.7 Skip to main content link is not visible when focused, and it does not work

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1968]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/en/ and all the sub pages

## Description of the problem
The service has a skip to main content -link. The link gets a keyboard focus, but the link doesn't appear at all from the keyboard focus. However, the NVDA screen reader can find the link. The name of the link is in English in all language versions; "Skip to main". The link doesn't work. Activating the link with the enter or space keys does not move the focus.
How to test: Use only the Tab key to move around. When you are on one of the finto pages and the browser's cursor bar is active, press tab-key.

**Expected result:** Skip to main content link is focused and the link becomes visible. Link name is in Finnish on the Finnish site, in Swedish on the Swedish site etc. Activating the link (enter) moves the keyboard focus to the main content (typically to h1 heading). After that the next tab press moves the focus to the next focusable element (eg.on first page it is Allärs - General thesaurus in Swedish -link).

**Fix:** Make skip link visible when focused, label the link descriptively according to the selected main language. Make sure that the link moves the keyboard focus to the main content, typically to the h1 heading. Make sure there is a skip link on every page.

Severity: moderate
Priority: medium

## Additional information (e.g. screenshots) about the problem

<img width="713" height="439" alt="Image" src="https://github.com/user-attachments/assets/79ff7914-6cbc-44d0-926f-ccedf4621a77" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
The issue is not fixed yet.
